webui/react/src/pages/Settings/GroupManagement.tsx

```diff
@@ -84,6 +90,7 @@ const GroupActionDropdown = ({
 const GroupManagement: React.FC = () => {
   const [ groups, setGroups ] = useState<V1GroupSearchResult[]>([]);
   const [ groupUsers, setGroupUsers ] = useState<V1GroupDetails[]>([]);
+  const [ roles, setRoles ] = useState<UserRole[]>([]);
```

Comment:
Can you put this in the global Store instead?

Reply:
✅